Resources/doc/reference/form_types.rst

```diff
@@ -370,6 +370,9 @@ According the choice made only associated fields are displayed. The others field
 map
   Associative array. Describes the fields that are displayed for each choice.
 
+**TIP**: To mask a subfield replace `.` by `__`.
```

Comment:
So now back to your underscore `__` tip. What does it do? Where should it be used? Can you give a code sample?